docs/fsharp/tour.md

```diff
@@ -1,3 +1,17 @@
+---
+title: Tour of F#
+description: Tour of F# 
+keywords: .NET, .NET Core, F#, Tour
+author: cartermp
+manager: wpickett
+ms.date: 08/18/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 49775139-082e-442f-b5a2-dd402399b5d2
+---
```

Comment:
`ms.prod` being `visual-studio-dev14` doesn't make much sense here.  F# isn't tied to VS.  I'm thinking of keeping it as-is.